src/Sarif.WorkItems/SarifWorkItemFiler.cs

```diff
@@ -26,15 +49,20 @@ public class SarifWorkItemFiler
         /// current implementation, this context is copied for each SARIF file (if any) split
         /// from the input log and then further elaborated upon.
         /// </param>
-        public SarifWorkItemFiler(SarifWorkItemContext filingContext)
+        public SarifWorkItemFiler(SarifWorkItemContext filingContext) : this(filingContext.HostUri)
```

Comment:
>this [](start = 72, length = 4)

Why are we calling the ctor above, when we're just going to undo everything it did (setting FilingContext and filingClient)?